### PR TITLE
Added covid policies to the landing

### PR DESCRIPTION
**Pull request by EncryptEx**
Card image and colors are missing.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -26,25 +26,25 @@
...
     <!-- <meta property="og:video:secure_url" content="https://www.youtube.com/watch?v=qTijleLsync"> -->
 
     <meta name="twitter:card" content="summary_large_image" />
     <meta name="twitter:site" content="@hackupc" />
     <meta name="twitter:creator" content="@hackupc" />
-    <meta name="twitter:title" content="HackUPC 2021 | May 14&ndash;16  | Online hackathon Barcelona" />
-    <meta name="twitter:description" content="Online hackathon in Barcelona on May 14&ndash;16 for 36 horus" />
-    <meta name="twitter:image" content="https://hackupc.com/ogimage.png?v=2021-v1" />
-    <meta name="twitter:image:secure_url" content="https://hackupc.com/ogimage.png?v=2021-v1" />
+    <meta name="twitter:title" content="HackUPC 2022 | April 29&ndash;1 May  | Hybrid hackathon Barcelona" />
```
Comment (carlotacb) [inline src/index.html]: Be carefull! Every where you put 1 May and it's May 1

**Review by carlotacb — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (EncryptEx) [reply src/index.html]: Thanks, that's true

**Review by carlotacb — APPROVED** [review]

**Comment by EncryptEx** [comment]
Covid card icon missing, will place it later at branch 2022

**Comment by EncryptEx** [comment]
> Covid card icon missing, will place it later at branch 2022

Done

**Pull request merged**